hack/dind-cluster.sh

```diff
@@ -221,7 +221,7 @@ function wait-for-cluster() {
   oc="$(os::build::find-binary oc)"
 
   # wait for healthz to report ok before trying to get nodes
-  os::util::wait-for-condition "ok" "${oc} get --config=\"${kubeconfig}\" --raw=/healthz" "120"
```

Comment:
Ah, whatever checkout of Origin I'm in right now has the `os::util::wait-for-condition` function in `provision-util.sh`.